api/dbadapters/snowflake.ts

```diff
@@ -203,13 +203,18 @@ where LOWER(table_schema) != 'information_schema'
     options: { limit: number } = { limit: 1000 }
   ): Promise<dataform.ITableMetadata[]> {
     const results = await this.execute(
-      `select tables.table_catalog as table_catalog, tables.table_schema as table_schema, tables.table_name as table_name
-       from information_schema.tables as tables
-       left join information_schema.columns as columns on tables.table_catalog = columns.table_catalog and tables.table_schema = columns.table_schema
-         and tables.table_name = columns.table_name
-       where tables.table_catalog ilike :1 or tables.table_schema ilike :1 or tables.table_name ilike :1 or tables.comment ilike :1
-         or columns.column_name ilike :1 or columns.comment ilike :1
-       group by 1, 2, 3
+      `select * from (
+        select tables.table_catalog as table_catalog, tables.table_schema as table_schema, tables.table_name as table_name
+        from information_schema.tables as tables
+        left join information_schema.columns as columns on tables.table_catalog = columns.table_catalog and tables.table_schema = columns.table_schema
+          and tables.table_name = columns.table_name
+        where tables.table_catalog ilike :1 or tables.table_schema ilike :1 or tables.table_name ilike :1 or tables.comment ilike :1
+          or columns.column_name ilike :1 or columns.comment ilike :1
+          and LOWER(tables.table_schema) != 'information_schema' and LOWER(tables.table_schema) != 'pg_catalog' and LOWER(tables.table_schema) != 'pg_internal'
```

Comment:
hold up, this is weird - why are we excluding _postgres_ stuff in a snowflake DB?

Reply:
Nice catch! When I see something weird to do with Postgres I naturally assume it's exactly how it should be.

Looking around, I think that's not meant to be there; have taken it out of both list and search. I imagine the SQL for `tables()` was copied other from either redshift or postgres without removing this bit.